app/weather_parser.py

```diff
@@ -1,17 +1,23 @@
-import forecastio
+import json
+import requests
 from config import FORECAST_API_KEY
 
 
 class WeatherForecast:
 
-    def generate_forecast(self, latitude, longitude, frequency):
-        forecast = self.get_forecast_request(latitude, longitude)
-        #  ---------We won't support hourly and daily forecasts yet---------
-        return forecast.currently()
+    def connect(self, lat, lng, time=None):
```

Comment:
Please give this a better name. Its purpose is not just to *connect* but to retrieve API results, right?